OTRExporter/OTRExporter/Main.cpp

```diff
@@ -92,8 +134,12 @@ static void ExporterProgramEnd()
 		printf("Generating OTR Archive...\n");
 		otrArchive = LUS::Archive::CreateArchive(otrFileName, 40000);
 
+		printf("Adding version file.\n");
 		otrArchive->AddFile("version", (uintptr_t)versionStream->ToVector().data(), versionStream->GetLength());
 
+		printf("Adding sohVersion file.\n");
+		otrArchive->AddFile("sohVersion", (uintptr_t)sohVersionStream->ToVector().data(), sohVersionStream->GetLength());
```

Comment:
Storing as `sohVersion` since `version` is already taken to represent the rom's CRC value.

Alternatively we could use `programVersion` to be an agnostic name if we ever want to promote some of the logic into LUS.

Reply:
renamed to `portVersion` in anticipation of a MM port reusing the same code (regardless if it's promoted to LUS)